app/src/mixins/common-mixin.ts

```diff
@@ -247,4 +247,18 @@ export class CommonMixin extends Vue {
   scrollTo (id: string): void {
     return document.getElementById(id)?.scrollIntoView()
   }
+
+  /** Get a magic link.  */
+  magicLink (nr: any): string {
+    const registryHomeUrl = sessionStorage.getItem('REGISTRY_HOME_URL')
+    const accountId = JSON.parse(sessionStorage.getItem('CURRENT_ACCOUNT'))?.id || 0
```

Comment:
You should use a `+`  operator so that accountId is either a number or 0.

![image](https://github.com/user-attachments/assets/bc24e198-0775-4efc-a517-b7a2db4c5458)